core/src/cats/grant_bareos_privileges.in

```diff
@@ -52,7 +52,11 @@ then
 fi
 
 retval=0
-PGOPTIONS='--client-min-messages=warning' psql -f "${temp_sql_grants}" -d "${db_name}" || retval=$?
+if [[ "$(uname -s)" =~ _NT ]]; then
+  PGOPTIONS='--client-min-messages=warning' psql --no-psqlrc -f "$(cygpath -w ${temp_sql_grants})" -d "${db_name}" || retval=$?
+else
+  PGOPTIONS='--client-min-messages=warning' psql --no-psqlrc -f "${temp_sql_grants}" -d "${db_name}" || retval=$?
+fi
```

Comment:
We do not need the windows-native changes

Reply:
ok done